app/views/renewal/what_you_need.scala.html

```diff
@@ -37,7 +46,32 @@
         <li>@Messages("renewal.whatyouneed.line_1")</li>
         <li>@Messages("renewal.whatyouneed.line_2")</li>
         <li>@Messages("renewal.whatyouneed.line_3")</li>
-    </ul>
+
+    @activities.map { ba =>
+            @if(ba.hasBusinessOrAdditionalActivity(ArtMarketParticipant)) {<li>@Messages("renewal.whatyouneed.line_4")</li>}
+            @if(ba.hasBusinessOrAdditionalActivity(MoneyServiceBusiness)) {<li>@Messages("renewal.whatyouneed.line_5")</li>}
+            @msbActivities.map { msb =>
+                @if(ba.hasBusinessOrAdditionalActivity(MoneyServiceBusiness) && msb.msbServices.contains(TransmittingMoney)) {
+                    <li>@Messages("renewal.whatyouneed.line_6")</li>
+                    <li>@Messages("renewal.whatyouneed.line_7")</li>
+                    <li>@Messages("renewal.whatyouneed.line_8")</li>
+                }
+                @if(ba.hasBusinessOrAdditionalActivity(MoneyServiceBusiness) && msb.msbServices.contains(CurrencyExchange)) {
+                    <li>@Messages("renewal.whatyouneed.line_9")</li>
+                    <li>@Messages("renewal.whatyouneed.line_10")</li>
+                    <li>@Messages("renewal.whatyouneed.line_11")</li>
+                }
+                @if(ba.hasBusinessOrAdditionalActivity(MoneyServiceBusiness) && msb.msbServices.contains(ForeignExchange)) {
+                    <li>@Messages("renewal.whatyouneed.line_12")</li>
+                }
```

Comment:
Could be simplified? 

```
@if(ba.hasBusinessOrAdditionalActivity(MoneyServiceBusiness) {

               @if(msb.msbServices.contains(TransmittingMoney)) {
                    <li>@Messages("renewal.whatyouneed.line_6")</li>
                    <li>@Messages("renewal.whatyouneed.line_7")</li>
                    <li>@Messages("renewal.whatyouneed.line_8")</li>
                }

                @if(msb.msbServices.contains(CurrencyExchange)) {
                    <li>@Messages("renewal.whatyouneed.line_9")</li>
                    <li>@Messages("renewal.whatyouneed.line_10")</li>
                    <li>@Messages("renewal.whatyouneed.line_11")</li>
                }

                @if(msb.msbServices.contains(ForeignExchange)) {
                    <li>@Messages("renewal.whatyouneed.line_12")</li>
                }
}
```

Reply:
How?

Reply:
I have provided an example above. Only one call to hasBusinessOrAdditionalActivity and simplified `if` statement.